RULES.md

```diff
@@ -829,7 +837,7 @@ Trips should be referred to at least once in `stop_times.txt`.
 ##### References:
 * [Original Python validator implementation](https://github.com/google/transitfeed)
 
-#### WrongStopTimeStopLocationTypeNotice
+#### LocationWithUnexpectedStopTimeNotice
```

Comment:
It seems that the alphabetical order is not followed here, do you mind changing the location for this part of the doc please?

Reply:
Done.